docs/source/user_guide/poweruser.rst

```diff
@@ -6,3 +6,4 @@ The Daft Poweruser
     poweruser/memory
     poweruser/partitioning
     poweruser/distributed-computing
+    poweruser/daft-launcher
```

Comment:
Let's not make it its own section. I'd prefer I think:

1. Move `distributed-computing` out to its own section (outside of poweruser)
2. Add Daft Launcher as a subsection in distributed computing

Reply:
Will update.

Reply:
Updated.